experimental/dds/tree/api/tree.api.md

```diff
@@ -269,8 +269,6 @@ export abstract class GenericSharedTree<TChange> extends SharedObject<ISharedTre
     protected processCore(message: ISequencedDocumentMessage, local: boolean): void;
     // @internal
     processLocalEdit(edit: Edit<TChange>): void;
-    // (undocumented)
```

Comment:
does this file get regenerated when you revert it?  seems surprising the original change got away without updating this

Reply:
do you mean reverting `raiseContainerWarning` PR? That PR was reverted in main and this PR is against next

Reply:
i mean the original change that caused the api changes in this file (since it looks like you didn't touch tree.ts?)

Reply:
this file gets generated even after reverting it